src/encoded/static/components/item-pages/FileMicroscopyView.js

```diff
@@ -65,17 +64,12 @@ class FileMicroscopyViewOverview extends React.Component {
     render(){
         var { context, schemas, width, windowWidth } = this.props;
 
-        var setsByKey;
-        var table = null;
+        var setsByKey = expFxn.experimentSetsFromFile(context),
```

Comment:
Do you still need setsByKey? You replaced it with setUrls on line 71. If you don't need it anymore, it looks like you also don't need context.

Reply:
Nevermind, I just saw where it's used.

Reply:
I'll maybe try to change func `experimentSetsFromFile` to return a list instead of object keyed by `@id` if is not a big edit.